### DISCUSSION TITLE
Lesson 7: ERROR- Makefile:6: *** multiple target patterns.  Stop.

### DISCUSSION THREAD

**Discussion by krastanovv** [Cyfrin/foundry-full-course-cu#1367] [Q&A]
Hello, everyone!
I wrote everything exactly like Patrick, but I get the following error:
Makefile:6: *** multiple target patterns.  Stop.

Makefile
![Makefile](https://github.com/Cyfrin/foundry-full-course-f23/assets/123821714/19d5f9b0-34dd-4686-99e2-54d81bb544bc)

I can`t understand, what the problem is.
Help will be appreciated!

**Comment by 0xusmanf** [comment]
In your makefile the format isn't right, it has to be exactly like below.

```makefile
deploy:
	forge script script/DeployFundMe.s.sol:DeployFundMe --rpc-url $(SEPOLIA_RPC_URL) --private-key $(PRIVATE_KEY) --broadcast --verify --etherscan-api-key $(ETHERSCAN_API_KEY) -vvvv
```

**Reply by krastanovv** [reply]
What is different?

**Reply by 0xusmanf** [reply]
Formatting, you need to press tab before you start typing `forge script`

**Reply by krastanovv** [reply]
Amazing! Thank you so much!
